02-peer-protocol.md

```diff
@@ -856,6 +868,14 @@ seconds, and the protocol only supports an expiry in blocks.
 specification; larger amounts are not necessary, nor wise, during the
 bootstrap phase of the network.
 
+The node _responsible_ for paying the Bitcoin fee should maintain a small "fee
```

Comment:
What does the word 'small' add here?

Reply:
Good catch, it should be removed.

Reply:
Done in f5490f17d17ff49dc26ee459432b3c9db4fda8a9